eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -134,8 +134,8 @@ object Helpers {
     if (accept.dustLimitSatoshis > accept.channelReserveSatoshis) throw DustLimitTooLarge(accept.temporaryChannelId, accept.dustLimitSatoshis, accept.channelReserveSatoshis)
 
     // if minimum_depth is unreasonably large:
-    // MAY reject the channel.
-    if (accept.toSelfDelay > nodeParams.maxToLocalDelayBlocks) throw ToSelfDelayTooHigh(accept.temporaryChannelId, accept.toSelfDelay, nodeParams.maxToLocalDelayBlocks)
+    // MAY reject the channel. max CLTV is 2016
```

Comment:
same bikeshed comment as above
```suggestion
    // MAY reject the channel.
```